src/replication/DealReplicationWorker.ts

```diff
@@ -16,6 +17,8 @@ export default class DealReplicationWorker extends BaseService {
   private readonly workerId: string;
   private readonly lotusCMD: string;
   private readonly boostCMD: string;
+  // holds reference to all started crons to be updated
+  private cronRefArray: Map<string, [string, ScheduledTask]> = new Map<string, [string, ScheduledTask]>();
```

Comment:
Try https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements
so we get rid of magic number 0 and 1